WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteCardAndItem.kt

```diff
@@ -119,13 +119,24 @@ sealed class MySiteCardAndItem(open val type: Type, open val activeQuickStartIte
                         val visitors: UiString,
                         val likes: UiString,
                         val onCardClick: () -> Unit,
+                        val message: Text? = null,
                         val footerLink: FooterLink
                     ) : TodaysStatsCard(dashboardCardType = DashboardCardType.TODAYS_STATS_CARD)
 
                     data class FooterLink(
                         val label: UiString,
                         val onClick: () -> Unit
                     )
+
+                    data class Text(
```

Comment:
Nit pick: I know it's text, but `Text` feels very generic as the type for message. I don't have a better suggestion, just wanted to call it out so we can think about it.

Reply:
You're right, actually, I kept the naming closer to Stats -> BlockListItem.Text

https://github.com/wordpress-mobile/WordPress-Android/blob/a5568fac17d88b7d669c765a531758152ec56211/WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/BlockListItem.kt#L160

But we have a different structure. I'll rename it to `TextWithLinks` till we come up with a better name, at least it is better than generic name `Text`. 😄

Reply:
Done in [dc52482](https://github.com/wordpress-mobile/WordPress-Android/pull/16268/commits/dc52482f9f452b535681f1c160c4b3daeb403958). I'll merge the PR now.